caster/lib/ccr/core/nav.py

```diff
@@ -51,6 +53,14 @@ class NavigationNon(MappingRule):
 
         "refresh":                          R(Key("c-r"), rdescript="Refresh"),
         
+        "maxiwin":                          R(Key("w-up"), rdescript="Maximize Window"),
+        "move window":                      R(Key("a-space, r, a-space, m"), rdescript="Move Window"),
+        "window (left | lease) [<n>]":      R(Key("w-left"), rdescript="Window Left") * Repeat(extra="n"),
+        "window (right | ross) [<n>]":      R(Key("w-right"), rdescript="Window Right") * Repeat(extra="n"),
+        "monitor (left | lease) [<n>]":     R(Key("sw-left"), rdescript="Monitor Left") * Repeat(extra="n"),
+        "monitor (right | ross) [<n>]":     R(Key("sw-right"), rdescript="Monitor Right") * Repeat(extra="n"),
+        "switch (window | windows)":        R(Key("ca-tab"), rdescript="Switch Window") * Repeat(extra="n"),
```

Comment:
This command is what tie match / ex match are trying to accomplish. Oi. I wish I'd known about it. I've always just used alt-tab. :-/